File: 3 КУРС/2 семестр/Java/JavaProects/app/src/main/java/com/example/voiceassistent/Forecast.java
package com.example.voiceassistent;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Forecast implements Serializable {

    @SerializedName("current")
    @Expose
    public Weather current;

    public class Weather{
        @SerializedName("temperature")
        @Expose
        public Integer temperature;

        @SerializedName("weather_descriptions")
        @Expose
        public List<String> weather_descriptions;
    }
}

File: 3 КУРС/2 семестр/Учебная практика/asd/JavaProects/app/src/main/java/com/example/voiceassistent/Forecast.kt
package com.example.voiceassistent

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Forecast : Serializable {
    @SerializedName("current")
    @Expose
    var current: Weather? = null

    inner class Weather {

        @SerializedName("temperature")
        @Expose
        var temperature: Int? = null

        @SerializedName("weather_descriptions")
        @Expose
        var weather_descriptions: List<String>? = null
    }
}